### DISCUSSION TITLE
X1nput v3.0.1

### DISCUSSION THREAD

**Discussion by araghon007** [araghon007/X1nput#25] [General]
Fixed the HID report structure, which seems to work with my bluetooth setup. Hopefully it'll work with yours too.

<hr /><em>This discussion was created from the release <a href='https://github.com/araghon007/X1nput/releases/tag/v3.0.1'>X1nput v3.0.1</a>.</em>

**Comment by marterikd** [comment]
hi, is there a way to "auto start" it when i open the game (dirt rally 2.0)? or is there a way to make it permanent? when i close and restart the game the rumble goes away. is there a way to "integrate" it into the game so i don't have to do setup every time i play?

**Reply by araghon007** [reply]
Not at the moment, sorry

**Reply by marterikd** [reply]
i understand. thanks for the reply
